src/plugins/toml/README.md

```diff
@@ -0,0 +1,43 @@
+- infos = Information about the toml plugin is in keys below
+- infos/author = Jakob Fischer <jakobfischer93@gmail.com>
+- infos/licence = BSD
+- infos/provides = storage/toml
+- infos/needs =
+- infos/recommends = type
+- infos/placements = getstorage setstorage
+- infos/status = experimental unfinished nodoc
+- infos/metadata = order comment/# comment/#/start comment/#/space type
+- infos/description = This storage plugin reads and writes TOML files, using Flex and Bison.
+
+# TODO: Documentation
+
+## NULL keys
```

Comment:
It might make sense to use the [`null` plugin](https://www.libelektra.org/plugins/null) to store these types of key values instead.